### ISSUE TITLE
streamdeck:keypress:up & down event not working.

### ISSUE THREAD

**Issue by Tylerjet** [SReject/easy-streamdeck#1]
Hey Man you are awesome for making this i finally got a issue with saving settings working because of you :) but one issue i found is that the on keypresses do not register for some reason. i tried to put a basic console log to register that the button was pressed and nothing outputs.
```javascript
streamdeck.on('streamdeck:keypress:up', function (evt) {
    	let context = evt.data;
    	console.log("button press");
    });
```

**Comment by Tylerjet** [comment]
Nvm i went through the js file and noticed that it was no longer keypress and just key. ```streamdeck:key:up``` did work.

**Comment by SReject** [comment]
Thats a bug on my part; it **SHOULD** be `streamdeck:keypress`

**Comment by SReject** [comment]
Fixed as of a3533ab83a6377da2c4808d9434c1d81c1296631
